package/kernel/mac80211/files/lib/wifi/mac80211.sh

```diff
@@ -77,9 +93,21 @@ detect_mac80211() {
 		[ "$found" -gt 0 ] && continue
 
 		mode_band="g"
-		channel="11"
+		channel="6"
```

Comment:
Though 11 isn't nice as well, I'd really like to see a sane default channel for ETSI regulatory domain.
While in FCC lands using 1, 6, 11 are the only non-overlapping channels, in ETSI lands we can use 1, 5, 9, 13 and turn off legacy 802.11b rates (so that channel width is really just 20MHz and not 22MHz for CCK rates).
Hence I suggest to just use '5' by default (which may harm legacy setups) instead of '6' (which harms non-legacy setups).

Reply:
Done

Reply:
5 is a really bad idea due to the problems you've described. If you want the router to behave fair (without potentially causing massive interference) in a default configuration, the only channel you should use is 1. It's up to the user to decide on using FCC or ETSI channels.